src/toady/cli.py

```
@@ -0,0 +1,57 @@
+)
+@click.option("--undo", is_flag=True, help="Unresolve the thread instead of resolving")
+def resolve(thread_id: str, undo: bool) -> None:
+    """Mark a review thread as resolved or unresolved."""
+    action = "Unresolving" if undo else "Resolving"
+    click.echo(f"{action} thread {thread_id}")
+    # TODO: Implement resolve logic
+
+
+def main() -> None:
+    """Entry point for the CLI."""
+    cli(prog_name="toady")
```

Pylint (3.3.7)

[error] No value for argument 'ctx' in function call

(E1120)